web/pages/tasks/[urlSlug].tsx

```diff
@@ -23,7 +26,40 @@ interface Props {
 const TaskPage = (props: Props): ReactElement => {
   useAuthProtectedPage();
 
+  const router = useRouter();
+
   const { roadmap } = useRoadmap();
+  const arrayOfTasks = getUrlSlugs(roadmap);
+  const currentUrlSlugIndex = arrayOfTasks.indexOf(props.task.urlSlug);
```

Comment:
this isn't really a review comment (ie, it shouldn't block merging this story) but these lines make me wonder if we should be using `useMemo` for these 4 lines so that we're not redoing that reduce (and the indexOf) every time the page re-renders